plugins/modules/vmware_host_user_manager.py

```diff
@@ -0,0 +1,264 @@
+        """
+        user_spec = vim.host.LocalAccountManager.AccountSpecification(
+            id=self.user_name,
+            description=self.user_description
+        )
+
+        if self.user_password and self.override_user_password:
+            user_spec.password = self.user_password
+
+        try:
+            self.host_obj.configManager.accountManager.UpdateUser(user_spec)
+        except Exception as e:
+            self.module.fail_json(msg="Failed to update a new password: %s" % to_text(e))
+
+    def remove_user(self):
+        """
+        Remove a user
+        """
+        try:
+            self.host_obj.configManager.accountManager.RemoveUser(self.user_name)
+        except Exception as e:
+            self.module.fail_json(msg="Failed to remove a user: %s" % to_text(e.msg))
+
+    def execute(self):
+        # The host name is unique in vCenter, so find the host from the whole.
+        self.host_obj = self.find_hostsystem_by_name(self.esxi_hostname)
+        if self.host_obj is None:
+            self.module.fail_json(msg="Cannot find the specified ESXi host: %s" % self.params['esxi_hostname'])
+
+        # Search the specified user
+        user_account = self.search_user()
+
+        changed = False
+        msg = "The change will not occur for the user information."
+        if self.state == "present":
+            if user_account:
+                user_diff = self.ensure_user_info_diff(user_account[0])
+                # If you want to update a user password, require the override_user_passwd is true.
+                if user_diff or self.override_user_password:
+                    changed = True
+                    if self.module.check_mode:
+                        msg = "The user information will be updated."
+                    else:
+                        msg = "Updated the user information."
+                        self.update_user()
+            else:
+                changed = True
+                if self.module.check_mode:
+                    msg = "The new user will be added."
+                else:
+                    msg = "Added the new user."
+                    self.add_user()
+
+        if self.state == "absent":
+            if user_account:
+                changed = True
+                if self.module.check_mode:
+                    msg = "The user will be removed."
+                else:
+                    msg = "Removed the user."
+                    self.remove_user()
+
+        self.module.exit_json(changed=changed, msg=msg)
+
+
+def main():
+    argument_spec = vmware_argument_spec()
+    argument_spec.update(
+        esxi_hostname=dict(type="str", required=True),
+        user_name=dict(type="str", required=True, aliases=["local_user_name"]),
+        user_password=dict(type="str", aliases=["local_user_password"], no_log=True),
+        user_description=dict(type="str", aliases=["local_user_description"]),
+        override_user_password=dict(type="bool", default=False),
```

Comment:
```suggestion
        override_user_password=dict(type="bool", default=False, no_log=False),
```

Ansible heuristically checks if the user-provided input from the module is sensitive information or not. It is done by checking keywords like `password`. But `override_user_password` is not sensitive. So I think we should explicitly set `no_log=False` to avoid this false-positive warning:

```
[WARNING]: Module did not set no_log for override_user_password
```


This would only confuse the users.